jetty-http2/http2-http-client-transport/src/main/java/org/eclipse/jetty/http2/client/http/HttpConnectionOverHTTP2.java

```diff
@@ -52,6 +53,7 @@
     private final AtomicInteger sweeps = new AtomicInteger();
     private final Session session;
     private boolean recycleHttpChannels = true;
+    private int maxMultiplex = 1;
```

Comment:
This is duplicated state... which is almost always bad.  See below:

Reply:
Argh!!! it id worse!!! This is triple state as it is also stored in [Http2Session](https://github.com/eclipse/jetty.project/blob/jetty-10.0.6/jetty-http2/http2-common/src/main/java/org/eclipse/jetty/http2/HTTP2Session.java#L101-L102), but at least there is is kept correctly as the local and remote value.